README.md

```diff
@@ -101,14 +132,102 @@ credentials, ie:
 }
 ```
 
-## Publishing
+The contents of this file can be found in lastpass with the name
+_marketplace builder credentials_ and the data_bag should _never_ be committed into
+source control.
+
+## Build Marketplace Images
+
+Test Kitchen has several suites for publishing different combinations of images.
+
+If you're targeting a specific release for a cloud and/or product set you'll
+probably want to use a targeted combination.
+
+Run kitchen list to view all the release targets:
+
+```shell
+kitchen list
+```
 
 Run the kitchen suite for the products that you wish to build, eg:
 
 ```shell
 kitchen converge aws-all-publish
 ```
 
+## Test Marketplace Images
+
+Currently there is no automated acceptance testing of marketplace images.
+In general what we've done is launch one of more of the newly created images
+and follow the setup [AWS](docs.chef.io/aws_marketplace.html) or
+[Azure](https://docs.chef.io/azure_portal.html#azure-marketplace) setup
+documentation to make sure that the user can login and start using the software.
+
```

Comment:
Let's add a TODO note here about automated acceptance testing with a pedant run for a future task.